portfolio.py

```diff
@@ -106,48 +181,8 @@ def average_buyin(self, buys: list, sells: list):
 
         return count, bought_at
 
-    # download all ticker data in a single request
-    # harder to parse but this provides a signficant performance boost
-    def download_market_data(self, args, stocks):
```

Comment:
the merge was a bit weird but this method is still the same, its just lower in the file